### ISSUE TITLE
"Unbound variable" error message could suggest checking file path inputs

### ISSUE THREAD

**Issue by justinwoo** [dhall-lang/dhall-haskell#514]
I think "unbound variable" errors for trying to pass in a literal file path without the leader `./` is a pretty easy way to lose time being confused. I'd like to add some small message to suggest doing this check, but was wondering if anyone opposed this for some reason.

**Comment by Gabriel439** [comment]
Yeah, that seems reasonable to me

**Comment by Gabriel439** [comment]
It looks like this was already fixed by https://github.com/dhall-lang/dhall-haskell/pull/472:

```diff
commit 51add692624ea9917c6c9434e95ad87906111cd6
Author: Gabriel Gonzalez <Gabriel439@gmail.com>
Date:   Fri Jun 15 08:49:04 2018 -0700

    Improve error message for a path missing leading `./` (#472)
    
    Fixes #445

diff --git a/src/Dhall/TypeCheck.hs b/src/Dhall/TypeCheck.hs
index 2541456..ab4aaf7 100644
--- a/src/Dhall/TypeCheck.hs
+++ b/src/Dhall/TypeCheck.hs
@@ -966,7 +966,17 @@ prettyTypeMessage (UnboundVariable _) = ErrorMessages {..}
         \    │  (x : Bool) → Bool │                                                      \n\
         \    └────────────────────┘                                                      \n\
         \      ⇧                                                                         \n\
-        \      A ❰∀❱ or ❰forall❱ here would transform this into a valid function type    \n"
+        \      A ❰∀❱ or ❰forall❱ here would transform this into a valid function type    \n\
+        \                                                                                \n\
+        \                                                                                \n\
+        \● You forgot to prefix a file path with ❰./❱:                                   \n\
+        \                                                                                \n\
+        \                                                                                \n\
+        \    ┌────────────────────┐                                                      \n\
+        \    │ path/to/file.dhall │                                                      \n\
+        \    └────────────────────┘                                                      \n\
+        \      ⇧                                                                         \n\
+        \      This should be ❰./path/to/file.dhall❱                                     \n"
 
 prettyTypeMessage (InvalidInputType expr) = ErrorMessages {..}
   where
```
